src/Parser/Priest/Discipline/Modules/Features/AtonementHealingBreakdown.js

```diff
@@ -52,11 +52,11 @@ class AtonementHealingBreakdown extends React.Component {
           {bySource && Object.keys(bySource)
             .sort((a, b) => bySource[b].healing.effective - bySource[a].healing.effective)
             .map((spellId) => {
-              const { ability, healing } = bySource[spellId];
+              const { ability, healing, bolts } = bySource[spellId];
 
               const performanceBarPercentage = healing.effective / highestHealing;
 
-              return (
+              return ([
```

Comment:
Please use the Wrapper component in `common/Wrapper` instead of returning an array.

Reply:
I tried working with Wrappers but because these were <tr> it would break the display when I tried adding them in. Am I missing something that would let me do it easyly?

Reply:
The Wrapper component doesn't add any elements to the DOM so should be fine in a table.

Reply:
If I replace [ by <wrapper> and ] by </wrapper> and if I surround the 2nd block with {}, I get this 
![image](https://user-images.githubusercontent.com/25039288/33740426-a530ab96-db6e-11e7-8184-0c007bd66864.png)

Reply:
It needs to be capitalized, and you need to import the component from `common/Wrapper`. See other modules for examples.